app.py

```
@@ -1,80 +1,131 @@
     ],
-    "skill": [
-        Skill("Python",
-              "1-2 Years",
-              "example-logo.png")
-    ]
+    "skill": [Skill("Python", "1-2 Years", "example-logo.png")],
+    "user": [User("John Doe", "+447123456789", "johndoe@gmail.com")],
 }
 
 
-@app.route('/test')
+@app.route("/test")
 def hello_world():
-    '''
+    """
     Returns a JSON test message
-    '''
+    """
     return jsonify({"message": "Hello, World!"})
 
 
-@app.route('/resume/experience', methods=['GET', 'POST'])
+@app.route("/resume/experience", methods=["GET", "POST"])
 def experience():
-    '''
+    """
     Handle experience requests
-    '''
-    if request.method == 'GET':
+    """
+    if request.method == "GET":
         return jsonify()
 
-    if request.method == 'POST':
+    if request.method == "POST":
         return jsonify({})
 
     return jsonify({})
 
-@app.route('/resume/education', methods=['GET', 'POST'])
+
+@app.route("/resume/education", methods=["GET", "POST"])
 def education():
-    '''
+    """
     Handles education requests
-    '''
-    if request.method == 'GET':
+    """
+    if request.method == "GET":
         return jsonify({})
 
-    if request.method == 'POST':
+    if request.method == "POST":
         return jsonify({})
 
     return jsonify({})
 
 
-@app.route('/resume/skill', methods=['GET', 'POST'])
+@app.route("/resume/skill", methods=["GET", "POST"])
 def skill():
-    '''
+    """
     Handles Skill requests
-    '''
-    if request.method == 'GET':
+    """
+    if request.method == "GET":
         return jsonify({})
 
-    if request.method == 'POST':
+    if request.method == "POST":
         return jsonify({})
 
     return jsonify({})
+
+
+@app.route("/resume/user", methods=["GET", "POST"])
+def user():
+    """
+    Handles user requests
+    """
+    if request.method == "GET":
+        return {"user": data["user"]}
+
+    if request.method == "POST":
+        api_data = request.get_json()
+
+    if api_data is not None:
```

What if `api_data` is None? Can we return a meaningful error?

---

Handled